src/main/java/clients/ActivitiesClient.java

```diff
@@ -55,6 +55,60 @@ public Response postActivities(Activities activities) {
         return response;
     }
 
+    public Response postActivitiesWithWaitReason(ActivitiesPost activities) {
+
+        Response response;
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectNode node = objectMapper.valueToTree(activities);
+
+        if (activities != null) {
+            response = callPostActivities(node);
+
+            if (response.getStatusCode() == 401 || response.getStatusCode() == 403) {
+                saveUtils();
+                response = callPostActivities(node);
+            }
+
+        } else {
+            response = callPostActivities("{ }");
+
+            if (response.getStatusCode() == 401 || response.getStatusCode() == 403) {
```

Comment:
Would this be cleaner with 

if activities == null
  node = "{ }"

response = callPostActivities(node);

if (response.getStatusCode() == 401 || response.getStatusCode() == 403) {
     saveUtils();
     response = callPostActivities(node);
}